### PR TITLE
retrieve dynamically the score version in cli

### LINKED ISSUE #16: score version is hard coded in the CLI
look at class SlangCLI param SCORE_VERSION...

### PR DESCRIPTION
**Pull request by Bonczidai**
fixes #16

### CODE REVIEW

**Comment by meirwah** [comment]
Please review @avimora

**Comment by avimora** [comment]
The general idea is fine. You can use spring's propertyplaceholder to inject it directly to your bean with zero code.

**Comment by avimora** [comment]
I think slang version is more appropriate

**Comment by meirwah** [comment]
agreed

**Comment by meirwah** [comment]
@Bonczidai please look at spring's propertyplaceholder, maybe this can be a refernce : http://www.mkyong.com/spring/spring-propertyplaceholderconfigurer-example/

**Comment by avimora** [comment]
In context xml you should add `<context:property-placeholder location="application.properties"/>` and you can refer to it by `${score.version}` or whatever the property key is, and inject it directly to the bean. It can be done with xml or annotations (e.g. `@Value("${score.version}")`)

**Comment by Bonczidai** [comment]
updated - slang version is used, please review ..

File: score-lang-cli/src/main/java/org/openscore/lang/cli/SlangBanner.java [hunk 1]
```diff
@@ -38,13 +42,13 @@ public String getBanner() {
...
 		sb.append(System.lineSeparator());
-		sb.append(SlangCLI.getVersion());
+		sb.append(getVersion());
 		return sb.toString();
 	}
 
 	@Override
 	public String getVersion() {
-		return SlangCLI.getVersion();
+		return slangCLI.getSlangVersion();
```
Comment (meirwah) [inline score-lang-cli/src/main/java/org/openscore/lang/cli/SlangBanner.java]: why ?? why not add instead :
@Value("${slang.version}")
private String slangVersion;

Comment (avimora) [inline score-lang-cli/src/main/java/org/openscore/lang/cli/SlangBanner.java]: Agree

**Comment by meirwah** [comment]
@Bonczidai  can you look at the comments we made here?

**Comment by Bonczidai** [comment]
updated

**Comment by meirwah** [comment]
@avimora  please review...

**Comment by avimora** [comment]
gr8 :100: can I delete the branch after merge?

**Comment by meirwah** [comment]
yes

**Pull request merged**